src/js/components/Columns.js

```diff
@@ -104,13 +105,24 @@ export default class Columns extends Component {
   _getColumnBreakpoints () {
     // grab CSS styles from DOM after component mounted
     // default to small size ($size-small = 192px)
-    let minColumnWidth = 192;
     const container = findDOMNode(this.containerRef);
     const column = container.childNodes[0];
     const child = column.childNodes[0];
+    let minColumnWidth = 192;
+    let childStyles;
 
     if (child) {
-      const childStyles = window.getComputedStyle(child);
+      if (navigator && navigator.userAgent.indexOf('MSIE') !== -1
+        || navigator.appVersion.indexOf('Trident/') > 0) {
+        // IE11 takes longer to compute correct child width
+        clearTimeout(this._childStylesTimer);
+        this._childStylesTimer = setTimeout(() => {
+          childStyles = window.getComputedStyle(child);
+        }, 10);
+      } else {
+        childStyles = window.getComputedStyle(child);
+      }
+
       if (childStyles && childStyles.width) {
```

Comment:
It seems incorrect to use `childStyles` here in the timeout case since it won't be set yet.